src/Symfony/Component/ExpressionLanguage/ParsedExpression.php

```diff
@@ -35,11 +35,17 @@ public function __construct($expression, Node $nodes)
         $this->nodes = $nodes;
     }
 
+    /**
+     * @return Node
+     */
     public function getNodes()
     {
         return $this->nodes;
     }
 
+    /**
+     * @return string
+     */
```

Comment:
That's an example of the kind of phpdoc that is probably not useful enough to add.